scipy/optimize/tests/test_optimize.py

```diff
@@ -3255,3 +3256,31 @@ def sparse_rosen_hess(x):
     assert res_dense.nfev == res_sparse.nfev
     assert res_dense.njev == res_sparse.njev
     assert res_dense.nhev == res_sparse.nhev
+
+
+@pytest.mark.parametrize('workers', [None, 2])
+@pytest.mark.parametrize('method', ['l-bfgs-b'])
+class TestWorkers:
+
```

Comment:
Could you add a test that here the minimization yields the same results with and without parallelisation?